app/views/petitions/_open_petition_show.html.erb

```diff
@@ -15,10 +11,22 @@
   </details>
 <% end %>
 
-<%# Signable #%>
-<% if petition.can_be_signed? -%>
```

Comment:
Looks like `can_be_signed` is only being used in one other place now. It shouldn't be changed in this PR but we could probably drop it